tools/data_manipulation/read_netcdf/read_netcdf/netcdf_read.xml

```diff
@@ -0,0 +1,259 @@
+            &&
+            i=\$((\$i+1)) &&
+            python '$__tool_directory__/netcdf_read.py' '$input' '$var_tab' $var
+            #for $i,$uc in enumerate($user_choice)
+                #if $uc.condi_between.comparator=="bi"
+                    ${uc.dim} ${uc.condi_between.comparator} ${uc.condi_between.t1}-${uc.condi_between.t2}
+                #elif $uc.condi_between.comparator=="be"
+                    ${uc.dim} ${uc.condi_between.comparator} ${uc.condi_between.t1}-${uc.condi_between.t2}
+                #else
+                    ${uc.dim} ${uc.condi_between.comparator} ${uc.condi_between.value}
+                #end if
+            #end for   
+            '$condi_source_coord.lat_dim'
+            \$lat
+            '$condi_source_coord.lon_dim'
+            \$lon
+        
+            &&
+            cat 'header' | sed 's/array(\[//g' | sed 's/], dtype=float32)//g'| sed 's/,\s/_/g' | sed 's/(//g' | sed 's/)//g' > 'header_cleaned'
+            &&
+            cat 'header_cleaned' 'sortie.tabular' > 'supersortie.tabular'
+            &&
+            datamash transpose < 'supersortie.tabular' > 'supersortie_transposed.tabular'
+            &&
+            sed -i 's/_/\t/g' 'supersortie_transposed.tabular'
+            &&
+            cat 'header_names' 'supersortie_transposed.tabular' | sed 's/\s/\t/g' > 'output_dir/coord'\$i'.tabular'; 
+        done<'$coord_tabular'
+
+    #else
+
+        python '$__tool_directory__/netcdf_read.py' '$input' '$var_tab' $var
+        #for $i,$uc in enumerate($user_choice)
+            #if $uc.condi_between.comparator=="bi"
+                ${uc.dim} ${uc.condi_between.comparator} ${uc.condi_between.t1}-${uc.condi_between.t2}
+            #elif $uc.condi_between.comparator=="be"
+                ${uc.dim} ${uc.condi_between.comparator} ${uc.condi_between.t1}-${uc.condi_between.t2}
+            #else
+                ${uc.dim} ${uc.condi_between.comparator} ${uc.condi_between.value}
+            #end if
+        #end for
+        #if $condi_source_coord.condi_coord.coord=='yes_cust_coord'
+            $condi_source_coord.condi_coord.lat_dim $condi_source_coord.condi_coord.lat_val $condi_source_coord.condi_coord.lon_dim $condi_source_coord.condi_coord.lon_val
+        #end if
+        &&
+        cat 'header' | sed 's/array(\[//g' | sed 's/], dtype=float32)//g'| sed 's/,\s/_/g' | sed 's/(//g' | sed 's/)//g' > 'header_cleaned'
+        &&
+        cat 'header_cleaned' 'sortie.tabular' > 'supersortie.tabular'
+        &&
+        datamash transpose < 'supersortie.tabular' > 'supersortie_transposed.tabular'
+        &&
+        sed -i 's/_/\t/g' 'supersortie_transposed.tabular'
+        &&
+        cat 'header_names' 'supersortie_transposed.tabular' | sed 's/\s/\t/g' > 'final.tabular'
+
+    #end if
+
+
+    ]]></command>
+    <inputs>
+        <param type="data" name="input" label="Input netcdf file" format="netcdf,h5"/>
```

Comment:
only netcdf?

Reply:
yes, even if datatype is h5 (as netcdf is based on h5)

Reply:
what I meant is that you can remove h5 and only leave netcdf in

Reply:
but if the user don't change manually the datatype from h5 (automatic sniffing) to netcdf, the tool can't be used but in fact it works even if the datatype is h5